clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/CopyLastGoogleServerGroupAtomicOperation.groovy

```diff
@@ -128,6 +129,11 @@ class CopyLastGoogleServerGroupAtomicOperation extends GoogleAtomicOperation<Dep
         description.targetSize != null
         ? description.targetSize
         : ancestorServerGroup.capacity.desired
+    newDescription.distributionPolicy =
+      description.distributionPolicy != null ?
```

Comment:
Can't this also be an elvis?

Reply:
I think we want to explicitly null check since this is initialized with an empty list of zones in the UI.